Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/derivetime/TimeIntervalDeriver.kt

```diff
@@ -16,11 +16,14 @@ private fun alignToInterval(timestamp: Long) =
  * Derive CheckIn start and end times
  * @param startTimestampInSeconds [Long] timestamp in seconds
  * @param endTimestampInSeconds [Long] timestamp in seconds
+ *
+ * @return [Pair] of [Long] including derived startTime and endTime in seconds
```

Comment:
Not really related to this PR actually, but I think a data class with 2 properties as a return type is always better than a pair because the code then gets more self-explanatory and less error-prone.

Reply:
what would be the difference? `Pair` is data class actually. Since it is from Kotlin, I assume that usability is well described

Reply:
The main benefit is that in `CheckInsTransformer`the readability improves -  instead of: 

```
timesPair.first
timesPair.second
```
we would have something like
```
derivedTime.startTime
derivedTime.endTime
```

So its more self-explanatory and we also wouldn't really need the one line of documentation

Reply:
You convinced me , addressed below